utils/download_filtered.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const child_process = require('child_process');
+const async = require('async');
+const fs = require('fs-extra');
+const tmp = require('tmp');
+const _ = require('lodash');
+const logger = require('pelias-logger').get('download');
+
+function downloadFiltered(config, callback) {
+  const targetDir = config.imports.openaddresses.datapath;
+  const files = config.imports.openaddresses.files;
+
+  logger.info(`Attempting to download selected data files: ${files}`);
+
+  fs.ensureDir(targetDir, (err) => {
+    if (err) {
+      logger.error(`error making directory ${targetDir}`, err);
+      return callback(err);
+    }
+
+    async.each(files, downloadSource.bind(null, targetDir), callback);
+
+  });
+
+}
+
+function downloadSource(targetDir, file, callback) {
+  logger.info(`Downloading ${file}`);
+
+  // first convert the file path to the source name, like 'us/ny/city_of_new_york.csv' -> 'us-ny-city_of_new_york.zip'
```

Comment:
the code doesn't convert slashes, but it looks like that's actually the correct format for the URL, so this comment should be updated, right?